app/assets/stylesheets/_core.scss

```diff
@@ -41,10 +41,15 @@ header.page-header div { // stylelint-disable-line selector-no-qualifying-type
   }
 
   h1 {
-    @extend %govuk-heading-xl;
+    @include govuk-text-colour;
+    @include govuk-font($size: 48, $weight: bold);
+
```

Comment:
What's with the space between attributes here?

Reply:
pasted L44-L50 from `govuk-frontend` 😬 will sort them out before merging.